.github/workflows/azure-static-web-apps-green-pebble-01f5d5c1e.yml

```diff
@@ -57,6 +57,22 @@ jobs:
               (github.event_name == 'pull_request' && format('{0}-{1}', 'pr', github.event.pull_request.number)) ||
               github.ref_name
             }}
+          VITE_GITHUB_ACTOR: ${{ github.actor }}
```

Comment:
NB: @AlexAxthelm what will this render as on the DEV and PROD sites? Would it be the person that opens the `release` PR into `production??
Or would it just be "GH Actions trigger" or something similar?

Reply:
> The username of the user that triggered the initial workflow run. If the workflow run is a re-run, this value may differ from `github.triggering_actor`. Any workflow re-runs will use the privileges of `github.actor`, even if the actor initiating the re-run (`github.triggering_actor`) has different privileges. [Source](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs)

I think this is generally the user who made the triggering push (which is for our workflow, also who made the last commit almost always).

Reply:
> if you want to drop this line entirely, it's not a problem.
Not at all! honestly was just curious how it would render. 

Happy to merge as-is, and then if we come to realize some info isn't superfluous we can drop it later.